common/src/main/java/com/wynntils/models/territories/type/GuildResource.java

```diff
@@ -1,17 +1,17 @@
 /*
- * Copyright © Wynntils 2022-2023.
+ * Copyright © Wynntils 2022-2024.
  * This file is released under LGPLv3. See LICENSE for full license details.
  */
 package com.wynntils.models.territories.type;
 
 import net.minecraft.ChatFormatting;
 
 public enum GuildResource {
-    EMERALD(ChatFormatting.GREEN, "Emeralds", ""),
-    ORE(ChatFormatting.WHITE, "Ores", "Ⓑ"),
+    EMERALDS(ChatFormatting.GREEN, "Emeralds", ""),
+    ORE(ChatFormatting.WHITE, "Ore", "Ⓑ"),
     WOOD(ChatFormatting.GOLD, "Wood", "Ⓒ"),
-    FISH(ChatFormatting.AQUA, "Fishes", "Ⓚ"),
-    CROPS(ChatFormatting.YELLOW, "Crops", "Ⓙ");
+    FISH(ChatFormatting.AQUA, "Fish", "Ⓚ"),
+    CROP(ChatFormatting.YELLOW, "Crop", "Ⓙ");
```

Comment:
Being a guid noob, is this some well-established order of resources? (Seeing as you reordered it).

Reply:
Didn't reorder it, but just renamed the enums/strings so it matches ingame. Emeralds is plural, everything else is not. The order is the same as every in-game tooltip, but it has not changed.

Reply:
Ah, I see, the diff looked like a reordering. :) Should we have this alphabetical instead, or is the in-game tooltip to be considered canonical?

Reply:
> Ah, I see, the diff looked like a reordering. :) Should we have this alphabetical instead, or is the in-game tooltip to be considered canonical?

It's most definitely should be like the in-game tooltip, we use this for reconstructing territory-like tooltip in multiple places.